### PR TITLE
 Add EXT_disjoint_timer_query and GL_EXT_debug_marker to gles 

### PR DESCRIPTION
**Pull request by sotaroikeda**
It is for addressing servo/webrender#2127

---
This change is [Reviewable](https://reviewable.io/reviews/servo/gleam/135)

### CODE REVIEW

**Comment by sotaroikeda** [comment]
I am going to update pull request.

**Comment by sotaroikeda** [comment]
The pull request also adds GL_EXT_debug_marker to gles, since it is also supported by ANGLE now.

File: src/gles_fns.rs [hunk 1]
```diff
@@ -147,47 +147,102 @@ impl Gl for GlesFns {
 
     #[allow(unused_variables)]
     fn gen_queries(&self, n: GLsizei) -> Vec<GLuint> {
-        panic!("not supported")
+        if !self.ffi_gl_.GenQueriesEXT.is_loaded() {
+            return Vec::new();
+        }
+        unsafe {
+            let mut result = vec![0; n as usize];
+            self.ffi_gl_.GenQueriesEXT(n, result.as_mut_ptr());
```
Comment (kvark) [inline src/gles_fns.rs]: only this line needs to be inside `unsafe` block

File: src/gles_fns.rs [hunk 2]
```diff
@@ -147,47 +147,102 @@ impl Gl for GlesFns {
...
+            return result;
```
Comment (kvark) [inline src/gles_fns.rs]: no need to use `return` keyword

File: src/gles_fns.rs [hunk 3]
```diff
@@ -147,47 +147,102 @@ impl Gl for GlesFns {
...
 
     #[allow(unused_variables)]
     fn get_query_object_iv(&self, id: GLuint, pname: GLenum) -> i32 {
-        panic!("not supported")
+        if !self.ffi_gl_.GetQueryObjectivEXT.is_loaded() {
+            return 0;
+        }
+        unsafe {
+            let mut result = 0;
+            self.ffi_gl_.GetQueryObjectivEXT(id, pname, &mut result);
```
Comment (kvark) [inline src/gles_fns.rs]: same here

File: src/gles_fns.rs [hunk 4]
```diff
@@ -147,47 +147,102 @@ impl Gl for GlesFns {
...
+            result
+        }
     }
 
     #[allow(unused_variables)]
```
Comment (kvark) [inline src/gles_fns.rs]: let's remove all of the `unused_variables` of the affected functions now

File: src/gles_fns.rs (hunk 1 above)
Comment (sotaroikeda) [reply src/gles_fns.rs]: I will update it. I also going to update same one in gl_fns.rs.

File: src/gles_fns.rs (hunk 4 above)
Comment (sotaroikeda) [reply src/gles_fns.rs]: Oh, I forgot to remove it. I will remove them.

**Comment by sotaroikeda** [comment]
Applied the comments.

**Comment by sotaroikeda** [comment]
@kvark, can you review again?

**Comment by glennw** [comment]
Looks good, thanks!
 
(I'll publish once this merges).

@bors-servo r+

**Comment by glennw** [comment]
Published 0.4.15.

**Comment by sotaroikeda** [comment]
Thanks!

**Pull request merged**